### DISCUSSION TITLE
How to Update Chia from 1.1.4 to 1.1.5 Windows Version

### DISCUSSION THREAD

**Discussion by Helheimbaby** [Chia-Network/chia-blockchain#4423] [Q&A] [answered]
Hello Chia-Friends :)
Iam still new to Chia and startet with Chia 1.1.4 but now there is a newer Version 1.1.5 out and i dont know how to Update it in Windows.
Pls help an desperate Chia Friend :) thx

**Comment by lvcivs** [comment, marked as answer]
Download the new installer from here: https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#windows
Then stop the chia program and run the installer, that should do it.

**Reply by tvboxxxx** [reply]
Hi, i do like that, and now i need to resync... Maybe you know how can i update chia without loosing sync in future?

**Reply by lvcivs** [reply]
When you shut down your node for maintenane/upgrade, you will always miss a few blocks and will need to sync again. It should be quick, though. 

If you need to sync from zero, then something went wrong.
